Makefile.toml

```diff
@@ -252,7 +252,7 @@ args = ["clean"]
 [tasks.test]
 category = "Test"
 description = "Runs all tests"
-dependencies = ["test-rust", "test-swift"]
+dependencies = ["test-rust"]
```

Comment:
To be honest, I've not used `clean`, `clean-rust`, `test` or `test-rust` via `cargo make`. Unclear about `clean-swift`.